test/BUILD

```diff
@@ -23,6 +23,7 @@ java_library(
     deps = [
         "//:common",
         "@maven//:commons_io_commons_io",
+        "@maven//:info_picocli_picocli",
```

Comment:
We leverage picocli to parse command line arguments without having to write string parsing ourselves, which is error-prone and likely to be more difficult to maintain and extend.